src/ngraph/util.cpp

```diff
@@ -449,3 +450,10 @@ void ngraph::check_fp_values_isnan(const char* name, const double* array, size_t
         }
     }
 }
+
+AxisVector ngraph::get_default_order(size_t rank)
```

Comment:
You always call this on `shape.size()` so why not make it take `Shape` as its arg?
And maybe it should be `AxisVector AxisVector::default_order(Shape)`